.github/workflows/build-with-clang.yml

```
@@ -52,7 +53,7 @@ jobs:
       - name: Install mkl_random dependencies
         shell: bash -l {0}
         run: |
-          pip install cython setuptools pytest pytest-cov
+          pip install cython setuptools">=77"
           pip install ${{ matrix.numpy_build_version }}
```

you may want to rename the variable as well
```suggestion
          pip install ${{ matrix.numpy_version }}
```